### ISSUE TITLE
Task: Implement euclidean_norm in metrics.py

### ISSUE THREAD

**Issue by frey-perez** [childmindresearch/wristpy#22]
### Description

The goal is to implement a function that will calculate the value of ENMO (Euclidean Norm Minus One) from 3 dimensional accelerometer data. As the name suggests the metric is the Euclidean norm of the 3D acceleration signal minus 1g(1 standard gravity). As a measurement of physical activity, ENMO is one of the key metrics calculated by wristpy and ggir and is of high scientific interest in the fields of actigraphy and sleep research. 

The input of the function will be a Measurement object containing the acceleration data(an np array) and timestamps(a polars series) the function will output another Measurement object where the measurement will be the ENMO values at corresponding timestamps. 

### Tasks

- [x] The function should take as input an instance of the Measurement class, where the measurement attribute is acceleration in the form of an np.array and the time attribute is a polars Series of datetime.datetime objects. 
- [x] the function should output a Measurement object where the measurement attribute is enmo and the time attribute is the unmodified polars Series derived from acceleration.time.

**Comment by Asanto32** [comment]
Technically we don't need the time data for the ENMO calculation. Would only need it to get epoch level ENMO. Maybe rephrase to say time is unmodified (as a practice for task precision writing?)

**Comment by frey-perez** [comment]
> Technically we don't need the time data for the ENMO calculation. Would only need it to get epoch level ENMO. Maybe rephrase to say time is unmodified (as a practice for task precision writing?)

Good point! it was implicit in my head but I'll make that clear.
